README.markdown

```diff
@@ -2511,6 +2511,7 @@ Nginx API for Lua
 * [ngx.timer.at](#ngxtimerat)
 * [ngx.config.debug](#ngxconfigdebug)
 * [ngx.config.prefix](#ngxconfigprefix)
+* [ngx.config.workers](#ngxconfigworkers)
```

Comment:
This markdown file was automatically generated from `doc/HttpLuaModule.wiki` by the [ngx-releng](https://github.com/openresty/nginx-devel-utils/blob/master/ngx-releng) script. So we should really edit the latter instead.